docs/requirements.txt

```diff
@@ -7,3 +7,8 @@ pocean-core>=3
 pynco
 sphinx
 sphinx-autodoc-typehints
+pooch
+h5netcdf
+scipy
+nco
+numpy>=1.14,<2
```

Comment:
You don't need to repeat anything that is already a mandatory dependency. Those are already installed when we install the package.